src/components/PageNav/PageNav.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+type Link = {
+  isActive: boolean;
+};
+
+const getLinkClass = ({ isActive }: Link) => (
+  classNames('navbar-item', { 'has-background-grey-lighter': isActive })
+);
+
+export const PageNav = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          className={getLinkClass}
+          to="/"
+        >
+          Home
+        </NavLink>
+
+        <NavLink
+          className={getLinkClass}
+          to="/people"
```

Comment:
`people` and `home` are used for navigation as string literals (magic strings)  - consider creating constants for them